js/levels.js

```diff
@@ -8,12 +8,22 @@
 
 // createTestLevel is used in /js/gui.js ignoring the error
 // ignore JSHintBear
+
+
+function createTestLevel()
```

Comment:
So, this function will be overwritten by the function below and deleted during execution.